tests/performance/density.go

```diff
@@ -156,8 +155,8 @@ func runAudit(startTime time.Time, endTime time.Time) {
 // createBatchVMIWithRateControl creates a batch of vms concurrently, uses one goroutine for each creation.
 // between creations there is an interval for throughput control
 func createBatchVMIWithRateControl(virtClient kubecli.KubevirtClient, vmCount int) {
-	for i := 1; i <= vmCount; i++ {
-		vmi := createVMISpecWithResources(virtClient)
+	for i := 0; i < vmCount; i++ {
```

Comment:
This is a hidden change in a commit that says that it changes something else.
Please do this in a separate commit or PR. Please also reason for it when you do.